frontend/ui/widget/menu.lua

```diff
@@ -995,11 +995,23 @@ end
 
 function Menu:updatePageInfo(select_number)
     if #self.item_table > 0 then
-        if Device:hasDPad() then
+        local is_focused = self.itemnumber and self.itemnumber > 0
+        if is_focused or Device:hasDPad() then
+            self.itemnumber = nil -- focus only once
+            select_number = select_number or 1 -- default to select the first item
+            local x, y
+            local nb_cols = self.layout[1] and #self.layout[1] or 1
+            if nb_cols == 1 then
+                x = 1
+                y = select_number
+            else -- mosaic
+                x = select_number % nb_cols
+                y = (select_number - x) / nb_cols + 1
+            end
```

Comment:
Dunno how much I previously had to make Menu aware of the coverbrowser plugin (I think I didn't need anything), feels a bit sad to have to have it know about this case. But ok, it's quite limited.